content/activities/StreamingAPodcast.ts

```diff
@@ -0,0 +1,12 @@
+import Activitiy from '@/types/Activity'
+
+const StreamingAPodcast: Activitiy = {
+  title: 'Streaming A Podcast',
```

Comment:
こういうときのaは小文字でも大丈夫なはずです（[参考](https://www.thepodcasthost.com/planning/how-to-start-a-podcast/)）
```suggestion
  title: 'Streaming a Podcast',
```

お好みで

Reply:
あ、これミスですね！小文字にします！